README.md

```diff
@@ -315,6 +315,9 @@ manifests as it descends breadth first. You can see the generated manifests in
 `./generated/prod-azure`, which has the same logical directory structure as your
 deployment definition.
 
+**NOTE** If planning to use [Kustomize](https://kustomize.io/), use the flag
```

Comment:
IMHO the `kustomize` feature isn't a main line option on would use when trying to quickly get up to speed. In other words it might be throwing too much information at the user who is just trying to quickly get the idea of Fabrikate. This is an advanced feature that can live in the command documentation @evanlouie  thoughts?

Reply:
Agreed. Unless we do a complete re-write of the main README.md to better guide completely guide the user through using both helm and kustomize, I think we should give the feature a mention to say it exists (i.e. what you have on line 20) and give a full flushed out guide in a separate doc

Reply:
@andrewDoing leet's remove this